Announcements folder; create triage-wg-welcome

BatmanAoD opened the pull request:
First draft of triage wg announcement

@Dylan-DPC

BatmanAoD committed:
Create triage-wg-welcome.md

First draft of triage wg announcement

Centril commented on announcements/triage-wg-welcome.md:
```diff
@@ -0,0 +1,24 @@
+will alleviate a key pain point in Rust's growth.
+
+Triaging does not require reading or writing code. If you like the Rust project
+and would like to help it advance but do not feel comfortable submitting
```
Well if you are comfortable with submitting PRs you can and should still join... 😂

BatmanAoD replied:
Hm, I didn't mean for that to sound exclusionary, but you're right, it can be taken that way. I've removed that paragraph and added the "not require...code" bit to its own sentence in the second paragraph. Thanks!

emilyalbini commented:
r? @Mark-Simulacrum

BatmanAoD committed:
Update triage-wg-welcome.md

Change the "no programming" paragraph into a single sentence; realign raw text to limit at 80 columns

Dylan-DPC-zz commented on announcements/triage-wg-welcome.md:
```diff
@@ -0,0 +1,22 @@
+when scaling up development efforts, so the triage WG's efforts will alleviate
+a key pain point in Rust's growth.
+
+If you're interested in becomoing a triage WG member, please leave a message in
```
Typo in becoming

Dylan-DPC-zz commented on announcements/triage-wg-welcome.md:
```diff
@@ -0,0 +1,22 @@
+[rust-lang](https://github.com/rust-lang) repositories as well as for
+diagnosing failures in Crater runs.
+
+Triaging is a high-impact activity that requires a relatively small time
```
I'd rephrase that as "requires a relatively less time"

Dylan-DPC-zz reviewed (commented):
Except a few tidbits, looks good to me. You can correct them or else not an issue as I'll fix them when creating the post

Mark-Simulacrum commented on announcements/triage-wg-welcome.md:
```diff
@@ -0,0 +1,22 @@
+The Release team is pleased to announce the **triage working group**, which
+will assume responsibility for triaging issues and pull requests in the
```
I would note here that this is taking responsibility from the release team just to show who is currently in charge of this

Mark-Simulacrum commented on announcements/triage-wg-welcome.md:
```diff
@@ -0,0 +1,22 @@
+
+Triaging is a high-impact activity that requires a relatively small time
+commitment.  It also does not require reading or writing code.  The goal of
+triage is to prevent issues and pull requests from being ignored by the Rust
```
```suggestion
triage is to prevent issues and pull requests from being missed by the Rust
```

Mark-Simulacrum reviewed (approved):
Looks good overall. I would suggest posting this on users and internals whenever you're ready. We'll want to be sure to have a good process for people to follow before we do though -- maybe a signup sheet and an update to the directions on the forge incorporating triagebot (rustbot) commands for users without rust-lang/rust permissions.

If there's some updates to triagebot that are necessary or would make that easier I can spend some time adding those tomorrow.
